index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="sass/main.css">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="stylesheet" href="./webfontkit-20220518-164003/myfont.css">
+    <title>Book-Store</title>
+</head>
+
+<body>
+
+    <header>
+        <div class="pre-navbar bg-color">
+            <ul>
+                <li><img src="./img/Social1.png" alt=""></li>
+                <li><img src="./img/Social2.png" alt=""></li>
+                <li>SignIn</li>
+                <li>My Page</li>
+                <li>Language</li>
+            </ul>
+        </div>
+        <nav class="nav-bar">
+            <!-- Mobile-nav -->
+            <div class="mobile-nav">
+                <input id="menu-toggle" type="checkbox" />
+                <label class='menu-button menu-button-container' id="menuButton" for="menu-toggle">
+                </label>
+
+                <ul class="menu">
+                    <li><a class="menu-item" href="./About.html">About</a></li>
+                    <li><a class="menu-item" href="#">BuyNow</a></li>
+                    <li><a class="menu-item" href="#">Contact</a></li>
+                </ul>
+            </div>
+            <!-- End Mobile Nav -->
+
+            <!-- Start Desktop Nav -->
+            <div class="desktopnav">
+                <a href="./index.html"><img class="navbar-logo" src="./img/logo.png" alt="" width=130></a>
```

Comment:
- [ ] On the desktop version for the home page, kindly add some margin to the right and left of the entire page so that it will match the design template as much as possible
Your design
![Capture](https://user-images.githubusercontent.com/85095702/169388643-6e6549f7-c2f9-4505-a602-4b7f074c37a8.PNG)

Behance template
![figma](https://user-images.githubusercontent.com/85095702/169388709-34531bd8-d044-4e57-8c4d-dde9efb3507d.PNG)